### ISSUE TITLE
Feature Request - add ip alias in log.txt

### ISSUE THREAD

**Issue by racoencmderror** [R-Smith/vmPing#75]
Good afternoon,

please add a feature alias IP in the log.txt

before :
![before](https://user-images.githubusercontent.com/86336192/135998634-9ff04424-8e60-4cde-bdb9-c1f4f41fa1d2.png)

after : 
 
![after](https://user-images.githubusercontent.com/86336192/135998694-2e9246f7-4a6c-4b96-af53-b532318cc46c.png)

it will be better and neater when looking at the log

thank you very much.

**Comment by An-dir** [comment]
Hi, 
it's me again. was trying that in the past, and found my powershell script again.
Assuming you are in a powershell in the folder containing the vmPing.xml with your aliases and the vmping-status.txt with your status report then this script will give you a gui where you can filter your log. be patient if it's very long.


```
$output = @()
$iplist = ([xml](gc .\vmPing.xml -Raw)).vmping.aliases.alias

$statuslog = gc .\vmping-status.txt
foreach ($line in $statuslog) {

    $currentHost = ($line -split "\t")[1]
    $currentHostText = ($iplist | where {$_.host -eq $currentHost})."#text"
    $output += $line + "`t`"" + $currentHostText + "`"" | convertfrom-csv -delimiter "`t" -header "DateTime","Host","State","Text"
}
$output | Out-GridView
```

**Comment by R-Smith** [comment]
Added here 79efe5b7efd2c71868705e24334ff8384768fa97
Available in release 1.3.20. The new export button on the status history window also writes the alias field.

**Comment by racoencmderror** [comment]
![image](https://user-images.githubusercontent.com/86336192/157611683-db7ba52a-7856-4485-a3ac-a52cecfc6e8c.png)

thank you very much for the update given, this is very useful for me
